R/checkMissings.R

```diff
@@ -57,6 +70,36 @@ insert_string <- function(df, rows, col, string) {
   df
 }
 
+#' @describeIn checkMissings compare missing tags and values in a certain range
+#'@export
+checkMissingsByValues <- function(GADSdat, missingValues = -50:-99, addMissingCode = TRUE) {
+  UseMethod("checkMissingsByValues")
+}
+
+#'@export
+checkMissingsByValues.GADSdat <- function(GADSdat, missingValues = -50:-99, addMissingCode = TRUE) {
+  check_GADSdat(GADSdat)
+  labels <- GADSdat$labels
+
+  missCode_missing_rows <- which(labels$value %in% missingValues & (is.na(labels$missings) | labels$missings == "valid"))
+  missCode_toomuch_rows <- which(labels$missings == "miss" & labels$value %in% missingValues)
+
+  ## Which variables are affected, how many adjustments are performed
+  if(length(missCode_missing_rows) > 0) {
+    message("The following variables have values in the 'missingValues' range which are not coded as missing:\n",

```

Comment:
Nit: Maybe add an information that the missings are updated, if the argument was set to `TRUE`?